setup.cfg

```diff
@@ -19,6 +19,7 @@ test_suite = tests
 scripts =
     bin/localstack
     bin/localstack.bat
+    bin/localstack-supervisor.py
```

Comment:
this is a nice idea! maybe we can remove the `.py` since it's now a script? just a nit